### PR TITLE
FoBo updated to v1.5

### PR DESCRIPTION
**Pull request by karma4u101**
This PR updates FoBo and FoBo related settings.

- FoBo updated from v1.4 to v1.5
- FoBo init param for JQuery has been updated from v1.10.2 to v1.11.3
- FoBo init param for Bootstrap has been updated from v3.2.0 to v.3.3.6
- Styling of menu entry for user drop down menu has been improved. 

If no one objects or beats me to it I will (if I remember to) merge the changes sometime next week.

### CODE REVIEW

File: scala_211/lift_advanced_bs3/src/main/webapp/assets/css/app-bs-theme.css [hunk 1]
```diff
@@ -56,9 +56,10 @@ body { padding-top: 70px; }
 }
 
 .navbar-default .navbar-nav > .open > a, 
+.navbar-default .navbar-nav > .active > a,
 .navbar-default .navbar-nav > .open > a:hover, 
 .navbar-default .navbar-nav > .open > a:focus {
-    background-color: #003C6F;
+    background-image: linear-gradient(to bottom, #003C6F 0%, #003C6F 100%);/*#003C6F;*/
```
Comment (Shadowfiend) [inline scala_211/lift_advanced_bs3/src/main/webapp/assets/css/app-bs-theme.css]: Usually you also specify a background color for non-gradient supporting browsers:

```css
  background-color: #003C6F;
  background-image: linear-gradient...;
```

Up to you if you want to do that here.

**Comment by Shadowfiend** [comment]
Other than the above, looks good to me!

File: scala_211/lift_advanced_bs3/src/main/webapp/assets/css/app-bs-theme.css (hunk 1 above)
Comment (karma4u101) [inline scala_211/lift_advanced_bs3/src/main/webapp/assets/css/app-bs-theme.css]: yea that's a good point, I will add it back.

**Comment by Shadowfiend** [comment]
👌 looks good to me! Will try to merge by the end of the day unless I hear complaints :)

**Pull request merged**